qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -1528,19 +1523,10 @@ def _connect_signals(self):
         view.renderProcessTerminated.connect(
             self._on_render_process_terminated)
         view.iconChanged.connect(self.icon_changed)
-        # WORKAROUND for https://bugreports.qt.io/browse/QTBUG-65223
-        if qtutils.version_check('5.10', compiled=False):
-            page.loadProgress.connect(self._on_load_progress_workaround)
-            self._load_finished_fake.connect(self._on_history_trigger)
-            self._load_finished_fake.connect(self._restore_zoom)
-            self._load_finished_fake.connect(self._on_load_finished)
-            page.loadFinished.connect(self._on_load_finished_workaround)
-        else:
-            # for older Qt versions which break with the above
-            page.loadProgress.connect(self._on_load_progress)
```

Comment:
This actually connected `_on_load_progress` twice, as it already got connected above.